### PR TITLE
Documentation for `Axiom` subdirectory

### PR DESCRIPTION
**Pull request by MatthewDaggitt**
Comments welcome! I'm sure I've misused at least a couple of technical terms somewhere...

### CODE REVIEW

File: README/Axiom.agda [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+-- type `P` or `¬P`. This is clearly impossible without any knowledge
+-- of what proposition `P` is.
+
+-- The types for which `P` or `¬P` holds is called `Dec P` in the
+-- standard library (short for `Decidable`).
+
+open import Relation.Nullary using (Dec)
+
+-- The type of the proof of saying that excluded middle holds for
+-- all types at universe level 0ℓ is therefore:
```
Comment (fredrikNordvallForsberg) [inline README/Axiom.agda]:
```suggestion
-- all types at universe level ℓ is therefore:
```

File: README/Axiom.agda [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+-- and this type is exactly the one found in `Axiom.ExcludedMiddle`:
+
+open import Axiom.ExcludedMiddle
+
+-- There are two different ways that the axiom can be introduced into
+-- your Agda development. The first option is to postulate it:
+
+postulate excludedMiddle : ExcludedMiddle 0ℓ
+
+-- This has the advantage that it only neesd to be postulated once
```
Comment (fredrikNordvallForsberg) [inline README/Axiom.agda]:
```suggestion
-- This has the advantage that it only need to be postulated once
```

File: README.agda [hunk 3]
```diff
@@ -43,40 +43,54 @@ module README where
 --     properties needed to specify these structures (associativity,
 --     commutativity, etc.), and operations on and proofs about the
 --     structures.
+
 -- • Axiom
---     The consequences of assuming various additional axioms
---     e.g. uniqueness of identity of proofs, function extensionality,
---     excluded middle.
+--     Types and consequences of various additional axioms not
+--     provable in Agda, e.g. uniqueness of identity of proofs,
```
Comment (fredrikNordvallForsberg) [inline README.agda]: Nitpicking, but UIP is of course provable in Agda, and function extensionality is provable using `--cubical`...
How about simply

Types and consequences of various additional axioms, e.g. uniqueness of identity of proofs,  function extensionality and excluded middle.

**Comment by MatthewDaggitt** [comment]
Great suggestions, thanks @fredrikNordvallForsberg!

**Comment by MatthewDaggitt** [comment]
As was pointed out in  #794 the modules themselves do all have a description of the axioms so I'm going to mark this as ready.

**Pull request merged**